### PR TITLE
Use networking.k8s.io/v1 for Ingress if possible

### LINKED ISSUE #639: [Helm] Ingress apiVersion to support "networking.k8s.io/v1"
The Ingress API has graduated from beta to a stable API since Kubernetes 1.19. The ingress apiVersion in helm charts needs to support "networking.k8s.io/v1".

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #639

### CODE REVIEW

File: helm/sealed-secrets/templates/ingress.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "sealed-secrets.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: networking.k8s.io/v1beta1
 {{- else -}}
 apiVersion: extensions/v1beta1
```
Comment (tewfik-ghariani) [inline helm/sealed-secrets/templates/ingress.yaml:9]: The `extensions/v1beta1` is also about to be deprecated

Ref : https://github.com/kubernetes/kubernetes/issues/94761

**Comment by tewfik-ghariani** [comment]
Any idea when will this get merged?

cc @mkmik

**Comment by farodin91** [comment]
Updated

**Comment by kobejn-jb** [comment]
I'm using this chart and having this released would be really useful.
Can some maintainer review this?

**Comment by davidkarlsen** [comment]
@mkmik PTAL? This will be required for running on k8s 1.22

**Comment by juan131** [comment]
We're performed a major refactorization of the chart at #687 today, see:
```
$ helm repo update sealed-secrets
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "sealed-secrets" chart repository
Update Complete. ⎈Happy Helming!⎈
$  helm search repo sealed-secrets/sealed-secrets
NAME                         	CHART VERSION	APP VERSION	DESCRIPTION
sealed-secrets/sealed-secrets	2.0.0        	v0.17.1    	Helm chart for the sealed-secrets controller.
```
This new chart version should already tackle this. Could you please give it a try?

**Comment by juan131** [comment]
I proceed to close this PR since it's already addressed at https://github.com/bitnami-labs/sealed-secrets/pull/687. Feel free to reopen if you consider this requires further discussion.

**Pull request closed without merging**